driver-core/src/main/java/com/datastax/driver/core/Responses.java

```diff
@@ -85,17 +85,17 @@ public DriverException asException(InetSocketAddress host) {
                 case SERVER_ERROR:     return new DriverInternalError(String.format("An unexpected error occurred server side on %s: %s", host, message));
                 case PROTOCOL_ERROR:   return new DriverInternalError("An unexpected protocol error occurred. This is a bug in this library, please report: " + message);
```

Comment:
I think in both the cases of PROTOCOL_ERROR and INVALID it could be useful to include the host, as it is possible that the client is technically doing something valid, but the server can't parse it because its in a bad state (for whatever reason).